erdpy/projects/report/options/report_option.py

```diff
@@ -0,0 +1,21 @@
+from abc import abstractmethod
+from pathlib import Path
+from typing import Any, Optional
+
+
+class ReportOption:
```

Comment:
Do you think you can add a short description for this (how it's used to alter the generation of the report etc.)?